packages/tools/pie-stylelint-config/LICENCE.md

```diff
@@ -0,0 +1,17 @@
+Apache License
+Version 2.0, January 2004
+http://www.apache.org/licenses/
+
+Copyright (c) Just Eat Holding Ltd
```

Comment:
I was looking at this yesterday – I think all our licenses should be `Copyright (c) Just Eat Takeaway` now.

We should change any going forward that we notice (but especially new ones we add)

Reply:
updated - let me know if it still needs the `Holding Ltd` part - wasn't sure from your comment!